### PR TITLE
Chore/benchmark cleanup

### PR DESCRIPTION
**Pull request by godenzim**

### CODE REVIEW

File: pallets/acurast/src/lib.rs [hunk 1]
```diff
@@ -419,6 +423,7 @@ pub mod pallet {
             let attestation = validate_and_extract_attestation::<T>(&who, &attestation_chain)?;
 
             if !T::KeyAttestationBarrier::accept_attestation_for_origin(&who, &attestation) {
+                #[cfg(not(feature = "runtime-benchmarks"))]
```
Comment (RomarQ) [inline pallets/acurast/src/lib.rs:426]: I think we could use a valid attestation in the benchmark

File: pallets/hyperdrive-outgoing/src/types.rs [hunk 2]
```diff
@@ -29,7 +29,6 @@ use serde::{Deserialize, Serialize};
 use sp_core::RuntimeDebug;
 use sp_runtime::traits;
 #[cfg(not(feature = "std"))]
```
Comment (gitsimon) [inline pallets/hyperdrive-outgoing/src/types.rs:31]: probably this exclusion is unnecessary, elsewhere it is not excluded under std

File: pallets/processor-manager/src/lib.rs [hunk 3]
```diff
@@ -203,13 +211,15 @@ pub mod pallet {
                 match update.operation {
                     ListUpdateOperation::Add => {
                         if !update.item.validate_timestamp::<T>() {
```
Comment (RomarQ) [inline pallets/processor-manager/src/lib.rs:213]: Can we instrument the block time for the benchmarks?

File: pallets/processor-manager/src/lib.rs [hunk 4]
```diff
@@ -203,13 +211,15 @@ pub mod pallet {
...
+                            #[cfg(not(feature = "runtime-benchmarks"))]
```
Comment (RomarQ) [inline pallets/processor-manager/src/lib.rs:214]: Can we instrument the block time for the benchmarks?

File: pallets/processor-manager/src/lib.rs [hunk 5]
```diff
@@ -203,13 +211,15 @@ pub mod pallet {
...
                             return Err(Error::<T>::PairingProofExpired)?;
                         }
                         let counter = Self::counter_for_manager(&who)
                             .unwrap_or(0u8.into())
                             .checked_add(&1u8.into())
                             .ok_or(Error::<T>::CounterOverflow)?;
                         if !update.item.validate_signature::<T>(&who, counter) {
+                            #[cfg(not(feature = "runtime-benchmarks"))]
```
Comment (RomarQ) [inline pallets/processor-manager/src/lib.rs:222]: Is this necessary?

**Review by gitsimon — CHANGES_REQUESTED** [review]

File: pallets/processor-manager/src/lib.rs [hunk 6]
```diff
@@ -235,6 +245,7 @@ pub mod pallet {
             let who = ensure_signed(origin)?;
 
             if !pairing.validate_timestamp::<T>() {
+                #[cfg(not(feature = "runtime-benchmarks"))]
```
Comment (RomarQ) [inline pallets/processor-manager/src/lib.rs:248]: Same as above

File: pallets/processor-manager/src/lib.rs [hunk 7]
```diff
@@ -252,6 +263,7 @@ pub mod pallet {
                 .ok_or(Error::<T>::CounterOverflow)?;
 
             if !pairing.validate_signature::<T>(&pairing.account, counter) {
+                #[cfg(not(feature = "runtime-benchmarks"))]
```
Comment (RomarQ) [inline pallets/processor-manager/src/lib.rs:266]: Same as above

**Review by RomarQ — APPROVED** [review]

**Review by gitsimon — APPROVED** [review]

**Pull request merged**